tasks/createTagsTasks.ts

```diff
@@ -130,25 +137,62 @@ async function tagRepoAsync(
         logError(`Failed to create tag for ${commit} in ${owner}/${repo}.`);
         return false;
     }
-    const refCreationResponse = await octokit.request(`Post /repos/${owner}/${repo}/git/refs`, {
-        owner: owner,
-        repo: repo,
-        ref: `refs/tags/${releaseTag}`,
-        sha: commit,
-    });
-
-    if (refCreationResponse.status !== 201) {
-        logError(`Failed to create reference for ${commit} in ${owner}/${repo}.`);
-        return false;
+    try {
+        const refCreationResponse = await octokit.request(`Post /repos/${owner}/${repo}/git/refs`, {
+            owner: owner,
+            repo: repo,
+            ref: `refs/tags/${releaseTag}`,
+            sha: commit,
+        });
+
+        if (refCreationResponse.status !== 201) {
+            logError(`Failed to create reference for ${commit} in ${owner}/${repo}.`);
+            return false;
+        }
+    } catch (err: any) {
+        if (err.status === 422 && err.message && err.message.includes('Reference already exists')) {
```

Comment:
allows the tagging to continue if tags already exists.  We sometimes need to do this if part of the tag creation fails (due to token expiry) and we want to re-run the job.